src/cluster/src/install.rs

```diff
@@ -555,6 +609,36 @@ impl ClusterInstaller {
         Ok(())
     }
 
+    /// Checks if all of the prerequisites for installing Fluvio are met
+    ///
+    /// This will attempt to automatically fix any missing prerequisites,
+    /// depending on the installer configuration. See the following options
+    /// for more details:
+    ///
+    /// - [`with_system_chart`]
+    /// - [`with_update_context`]
+    ///
+    /// [`with_system_chart`]: ./struct.ClusterInstaller.html#method.with_system_chart
+    /// [`with_update_context`]: ./struct.ClusterInstaller.html#method.with_update_context
+    fn check_and_fix(&self) -> Result<(), ClusterError> {
+        // Preforms pre-install checks and tries to fix things if possible
+        match self.pre_install_check() {
+            Ok(_) => (),
+            Err(ClusterError::MissingSystemChart) if self.config.install_sys => {
+                debug!("Fluvio system chart not installed. Attempting to install");
+                self._install_sys()?;
+            }
+            Err(ClusterError::InvalidMinikubeContext) if self.config.update_context => {
+                debug!("Updating to minikube context");
+                let context = MinikubeContext::try_from_system()?;
```

Comment:
we should ask from confirmation from user since this is not typical installation behavior for kubernetes

Reply:
This is gated behind the `update_context` configuration option. If you like, we can make that option default to `false` and enable it with a CLI flag as necessary

Reply:
sure